README.md

```diff
@@ -65,7 +65,7 @@ Usage:
 
 Options:
   -f, --format=FORMAT              Destination format [default: "openapi3.0.1"]
```

Comment:
Any reason why we can't have openapi3.1.0 as default?

Reply:
@prabhu I changed it to 3.0.1 because ZAP does not indicate supporting greater than 3.0 in their [docs](https://www.zaproxy.org/docs/desktop/addons/openapi-support/).

Reply:
Good point. Let's check with them.